cpp/include/raft/linalg/distance_type.h

```diff
@@ -35,18 +35,18 @@ enum DistanceType : unsigned short {
   EucUnexpandedL2Sqrt = 5,
   /** basic inner product **/
   InnerProduct = 6,
-  /** ChebyChev (Linf) distance **/
-  ChebyChev = 7,
+  /** Chebyshev (Linf) distance **/
+  UnexpandedLinf = 7,
```

Comment:
Sorry to keep the naming discussion ongoing... I'm not sure the unexpanded/expanded prefixes are necessary for the non-euclidean norms where we don't have both versions? E.g. we don't use it for InnerProduct. I'm not sure what an expanded Linf would look like either, so maybe we could omit those?

Reply:
The expanded/unexpanded just have to do with whether they can be derived from an inner product, not necessarily Euclidean. Linf & L1 don't have expanded forms as far as I know, but the current L1 distance is already marked as `EucUnexpandedL1` so I just kept that naming convention to keep things consistent. I can remove it from the Linf, but since the dense pairwise distance implementation uses the L1 enum, we would need to update that in cuml as well.